pkgs/top-level/aliases.nix

```diff
@@ -240,10 +240,12 @@ mapAliases ({
   ec2_api_tools = ec2-api-tools; # added 2021-10-08
   elasticmq = throw "elasticmq has been removed in favour of elasticmq-server-bin"; # added 2021-01-17
   elasticsearch7-oss = throw "elasticsearch7-oss has been removed, as the distribution is no longer provided by upstream. https://github.com/NixOS/nixpkgs/pull/114456"; # added 2021-06-09
-  electron_3 = throw "electron_3 has been removed in favor of newer versions"; # added 2021-01-06
-  electron_4 = throw "electron_4 has been removed in favor of newer versions"; # added 2021-01-06
-  electron_5 = throw "electron_5 has been removed in favor of newer versions"; # added 2021-01-06
-  electron_6 = throw "electron_6 has been removed in favor of newer versions"; # added 2021-01-06
+  electron_3 = throw "electron_3 has been removed in favor of newer versions"; # added 2022-01-06
```

Comment:
Why change the date of the aliases addition for electron_3 to _6 ?

Reply:
Fixing my own typo from 6730766. 2020 to 2022 has just blended together.

Reply:
Oh ok